src/restapi/mm_routes.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+
+import express, {Request, Response} from 'express';
+import {AppCall} from 'mattermost-redux/types/apps';
+
+import app from '../app/app';
+
+import {jsonStoreFileName} from '../app/constants';
+
+const router = express.Router();
+
+router.post('/createform', async (req: Request, res: Response) => {
+    const appCall: AppCall = req.body;
+    if (appCall.type === 'form') {
+        const createForm = app.calls.createForm(appCall.context.post.message);
```

Comment:
`app.calls.createForm` seems like a stretch for a pure function that has no state. Maybe do something like this

```suggestion
        const createTicketForm = newCreateTicketForm(appCall.context.post.message);
```

Reply:
Much cleaner. Done.